packages/suite/src/utils/firmware/index.ts

```diff
@@ -119,10 +119,17 @@ export const validateFirmware = (
     }
 };
 
-export const getFirmwareType = (device?: TrezorDevice) => {
+export const getSuiteFwTypeFromDevice = (device?: TrezorDevice) => {
     if (isDeviceInBootloaderMode(device)) {
         return '';
     }
 
-    return isBitcoinOnlyFirmware(device) ? FirmwareType.BitcoinOnly : FirmwareType.Universal;
+    return isBitcoinOnlyFirmware(device)
+        ? SuiteFirmwareType.BitcoinOnly
+        : SuiteFirmwareType.Universal;
 };
+
+export const getSuiteFwType = (firmwareType?: FirmwareType) =>
```

Comment:
why do we need the `SuiteFirmwareType` enum? 🤔

Reply:
It is just some translation thing `Regular` to `Universal`, I am not sure why we have it there. It probably makes sense to change it and remove it. I will try

Reply:
62b53e85588e893728993500fdfe9c5077e135f8
51fc01e1339629ef415f4e1e5c04fd3e42beb38b